Switched hh5 instances with xp65

atteggiani opened the pull request.

paigem reviewed (approved):
Looks great, thanks @atteggiani!

atteggiani commented:
A section on how to port suites to using `xp65` instead of `hh5` has been added in [58dd771](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/941/commits/58dd7718d13e78b9a2bff988aaeb24fc03822ea3).

paigem commented on docs/models/run_a_model/run_access-cm2.md:
```diff
@@ -599,11 +600,46 @@ For the `{{ suite_id }}` suite in this example, the `atm` directory contains:
+If your {{ model }} suite relies on `hh5`, update it by following the steps below:
+
+#### 1. Update storage flags {: .no-toc }
+In the `suite.rc` file, look for all `-l storage` instances and replace `hh5` occurrencies with `xp65`
```
Typo

```suggestion
In the `suite.rc` file, look for all `-l storage` instances and replace `hh5` occurrences with `xp65`
```

paigem reviewed (changes requested):
Thanks so much for getting to this @atteggiani! A few minor suggestions/questions before we are ready to merge.

And one bigger picture question about Run CM2 (admitting that I haven't read through the page in its entirety): we have a section about updating suites that still rely on hh5. Is there any clarity on when this would be the case or how someone would know if their suite still relies on hh5? Eg a specific error message? I assume a new user wouldn't have this issue (i.e. that the new suites in these instructions would not rely on hh5). But would all previous users have suites that rely on hh5?

atteggiani commented on docs/models/run_a_model/run_access-cm2.md:
````diff
@@ -599,11 +600,46 @@ For the `{{ suite_id }}` suite in this example, the `atm` directory contains:
+```
+
+!!! warning
+    The `conda/analysis3-25.05` version is the latest working version for {{ model }} setup. Therefore, we strongly recommend loading this version. For further information, refer to the [related Hive Forum post](https://forum.access-hive.org.au/t/issues-when-transitioning-from-hh5-to-xp65-in-suite-runs/4544).
````
I followed the instructions from the related Hive Post that mentioned an issue with ACCESS-CM2 suites and versions greater than `conda/analysis3-25.05`. 

However, the default example suite `u-cy339` [loads `conda/analysis3`](https://code.metoffice.gov.uk/trac/roses-u/browser/c/y/3/3/9/trunk/suite.rc#L493) without any version, and I think it works fine.
Therefore, the issue might have been fixed in later `conda/analysis3` versions. 

I think it's safe to change this to `conda/analysis3` and change the warning (saying that, for reproducibility, it might be best to select a specific version)

I'll change it

paigem commented on docs/models/run_a_model/run_access-cm2.md:
```diff
@@ -599,11 +600,46 @@ For the `{{ suite_id }}` suite in this example, the `atm` directory contains:
+### Update suites still relying on hh5
+Previously, the `hh5` NCI project hosted the Python environments and software often used within suites (e.g., `conda/analysis3` environemnts, `pythonlib/um2netcdf4` utility).<br>
+Due to `hh5` decommissioning, all references to `hh5` must be replaced with the project storing the updated versions of the environments and software. For suites derived from the example suite above (`{{ suite_id }}` ), the updated versions are stored in `xp65`.
+
```
```suggestion

If you're not sure whether your suite relies on `hh5`, you can run the following command to check for any `hh5` occurrences from within the suite directory: `grep -r hh5 --exclude-dir=.svn.` 

```

paigem commented on docs/models/run_a_model/run_access-cm2.md:
```diff
@@ -599,11 +600,46 @@ For the `{{ suite_id }}` suite in this example, the `atm` directory contains:
+Previously, the `hh5` NCI project hosted the Python environments and software often used within suites (e.g., `conda/analysis3` environemnts, `pythonlib/um2netcdf4` utility).<br>
+Due to `hh5` decommissioning, all references to `hh5` must be replaced with the project storing the updated versions of the environments and software. For suites derived from the example suite above (`{{ suite_id }}` ), the updated versions are stored in `xp65`.
+
+If your {{ model }} suite relies on `hh5`, update it by following the steps below:
```
```suggestion
If your {{ model }} suite does have instances of `hh5`, update it by following the steps below:
```

atteggiani commented:
@paigem 
I went on and updated the paragraph. I also did some minor phrasing updates and changed the position of the warnings to be at the beginning of the paragraphs. 
The changes are in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/941/changes/10c484063003ac413bce7c845698202adf0821fe

paigem reviewed (commented):
@atteggiani Thanks for your suggestion on how to know whether a suite relies on `hh5`. I've suggested one way we could add this suggestion in. I opted to not have this be Step 1, but rather just a tip beforehand.

paigem commented on docs/models/run_a_model/run_access-cm2.md:
```diff
@@ -602,40 +602,40 @@ Files formatted as `<suite-name>a.xhist-<year><month><day>` contain metadata inf
 
-#### 1. Update storage flags {: .no-toc }
-In the `suite.rc` file, look for all `-l storage` instances and replace `hh5` occurrences with `xp65`
+Then, follow the steps below to ensure your suite is not using `hh5`.
```
I suggest merging this sentence with the one you had below under the first step. I think this is clearer, unless I've misunderstood something.

```suggestion
If you see any output from the command above, follow the steps below to replace all `hh5` occurrences with `xp65`.
```

paigem reviewed (commented):
I like the updates you made in this commit @atteggiani! Just one suggestion to tidy up wording and make it more clear for readers.

atteggiani commented on docs/models/run_a_model/run_access-cm2.md:
````diff
@@ -602,40 +602,40 @@ Files formatted as `<suite-name>a.xhist-<year><month><day>` contain metadata inf
-module load conda/analysis3
-```
+#### 1. Replace hh5 with xp65 {: .no-toc }
+If you see any output from the command above, replace any `hh5` occurrency with `xp65`.<br>
````
Related comment above

paigem reviewed (approved):
Looks great, thanks @atteggiani!
